velox/serializers/PrestoSerializer.cpp

```diff
@@ -304,21 +321,45 @@ void readDecimalValues(
   }
 }
 
+vector_size_t sizeWithIncomingNulls(
```

Comment:
This function can be just `incomingNulls ? numIncomingNulls : size`, just inline it would be shorter